cypress/integration/data-import/view-all-filter.spec.js

```diff
@@ -0,0 +1,137 @@
+import DataImportViewAllPage from '../../support/fragments/data_import/dataImportViewAllPage';
+import fileManager from '../../support/utils/fileManager';
+import getRandomPostfix from '../../support/utils/stringTools';
+
+describe('ui-data-import: Filter the "View all" log screen', () => {
+  // Create unique name for a MARC file
+  const uniqueFileName = `test${getRandomPostfix()}.mrc`;
+  let query;
+
+  before(() => {
+    cy.login(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+
+    cy.getToken(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+
+    // create dynamically file with given name in fixtures
+    fileManager.createFile(`cypress/fixtures/${uniqueFileName}`);
+
+    // remove generated test file from fixtures after uploading
+    cy.uploadFile(uniqueFileName);
+    fileManager.deleteFile(`cypress/fixtures/${uniqueFileName}`);
+  });
+
+  it('C11113 Filter the "View all" log screen', () => {
+    DataImportViewAllPage.gotoViewAllPage();
```

Comment:
based on Testrail and "You'll go to a log summary screen with a list of all imports, in reverse chronological order" you need to make 2 imports and initially check the order of presented job results